utils/req-res-log-valiadtor.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python3
+import os
+import glob
+import json
+import jsonschema
+import subprocess
+import redis
+import time
+import argparse
+import multiprocessing
+import collections
+import io
+import signal
+import traceback
+from datetime import timedelta
+from functools import partial
+try:
+    from jsonschema import Draft201909Validator as schema_validator
+except ImportError:
+    from jsonschema import Draft7Validator as schema_validator
+
+"""
+The purpose of this file is to validate the reply_schema values of COMMAND DOCS.
+Basically, this is what it does:
+1. Goes over req-res files, generated by redis-servers, spawned by the testsuite (see logreqres.c)
+2. For each request-response pair, it validates the response against the request's reply_schema (obtained from COMMAND DOCS)
+
+In order to use this file you must run the redis testsuite with the following flags:
+./runtest --dont-clean --force-resp3 --log-req-res
+
+And then:
+./utils/req-res-log-validator.py
+"""
+
+
+class Request(object):
```

Comment:
please add some comments above big blocks of code in this file (classes, functions) to describe their role. e.g. in this case it's a request parser for the reqres file, right?